libs/global-contract/contract/src/lib.rs

```diff
@@ -66,15 +69,95 @@ impl GlobalData {
 
 #[near_bindgen]
 impl CommunityContract for GlobalData {
-    fn set_community_contract(&mut self, server: String, contract_address: AccountId) {
-        if !(self.admins.contains(&predecessor_account_id())) {
+    // fn set_community_contract(&mut self, server: String, contract_address: AccountId) {
+    //     if !(self.admins.contains(&predecessor_account_id())) {
+    //         throw_error(crate::errors::UNAUTHORIZED);
+    //     }
+    //     self.guild_id_to_community_contract
+    //         .insert(&server, &contract_address);
+    // }
+    // fn get_community_contract(&self, server: String) -> Option<AccountId> {
+    //     self.guild_id_to_community_contract.get(&server)
+    // }
+
+    fn init_community(&mut self, guild_id: String, new_admins: Vec<AccountId>) {
+        if !self.admins.contains(&predecessor_account_id()) {
+            throw_error(crate::errors::UNAUTHORIZED);
+        }
+        let mut admins = UnorderedSet::new(format!("community-contract-{}", guild_id).as_bytes());
+        for new_admin in new_admins.iter() {
+            admins.insert(new_admin);
```

Comment:
verify that `new_admin` is a valid accound ID via [`is_valid_account_id`](https://docs.rs/near-sdk/2.0.1/near_sdk/env/fn.is_valid_account_id.html)